src/wiki/keyboard-shortcuts.md

```diff
@@ -0,0 +1,208 @@
+| <kbd>Ctrl</kbd> + <kbd>Y</kbd>                                                              | Redo an action.                                                                                 |
+| <kbd>Ctrl</kbd> + <kbd>→</kbd>                                                              | Moves the cursor to the beginning of the next word.                                             |
+| <kbd>Ctrl</kbd> + <kbd>←</kbd>                                                              | Moves the cursor to the beginning of the previous word.                                         |
+| <kbd>Ctrl</kbd> + <kbd>↓</kbd>                                                              | Moves the cursor to the beginning of the next paragraph.                                        |
+| <kbd>Ctrl</kbd> + <kbd>↑</kbd>                                                              | Moves the cursor to the beginning of the previous paragraph.                                    |
+| <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Tab</kbd>                                           | Switch between opened apps with arrow keys. (You can let go of the <kbd>Alt</kbd> and tab keys) |
+| <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>↑</kbd> <kbd>↓</kbd> <kbd>→</kbd> <kbd>←</kbd>     | When a group or tile is selected on the Start menu, move it in the specified direction.         |
+| <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>↑</kbd> <kbd>↓</kbd> <kbd>→</kbd> <kbd>←</kbd>    | When a tile is selected on the Start menu, drag it onto another tile to create a folder.        |
+| <kbd>Ctrl</kbd> + <kbd>↑</kbd> <kbd>↓</kbd> <kbd>→</kbd> <kbd>←</kbd> + <kbd>Spacebar</kbd> | Select multiple individual items within a window or on the desktop.                             |
+| <kbd>Ctrl</kbd> + <kbd>Esc</kbd>                                                            | Opens Start Menu.                                                                               |
+| <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Esc</kbd>                                         | Opens Task Manager.                                                                             |
+| <kbd>Alt</kbd> + <kbd>Shift</kbd>                                                          | Changes the keyboard layout when multiple layouts are available.                                |
+| <kbd>Shift</kbd> + <kbd>F10</kbd>                                                           | Shows the context menu for the selected item.                                                   |
+| <kbd>Shift</kbd> with any arrow keys                                                        | Select multiple items within a window or on the desktop, or highlight text in a document.       |
+| <kbd>Shift</kbd> + <kbd>Del</kbd>                                                           | Permanently delete the selected item without sending it to the Recycle Bin.                     |
+
+# Windows 11 keyboard shortcuts
```

Comment:
```suggestion
## Windows 11 keyboard shortcuts
```

For better accessibility, only use H1s for the article title, and to make the "On this page" functionality working properly.